go/tools/builders/replicate.go

```diff
@@ -129,11 +131,6 @@ func replicateTree(src, dst string, config *replicateConfig) error {
 	if err := os.RemoveAll(dst); err != nil {
 		return fmt.Errorf("Failed to remove file at destination %s: %v", dst, err)
 	}
-	if l, err := filepath.EvalSymlinks(src); err != nil {
```

Comment:
CI is failing pretty badly and I don't understand all of the failures. If this is too weird, we could fix this for now by setting `GODEBUG=winsymlink=0` during compilation.

Reply:
I'll take a look at the failures today. `GODEBUG=winsymlink=0` might be a good backup option, as you say.

Incidentally, is there a reference for the CI images? I've sometimes found rules_go CI failures very hard to reproduce locally, especially on Windows.

Reply:
You could try to run https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/setup-windows.ps1 in a Windows VM.

Reply:
> You could try to run https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/setup-windows.ps1 in a Windows VM.

Thanks, that's probably what I need.